website/docs/reference/dbt-commands.md

```diff
@@ -9,17 +9,43 @@ You can run dbt using the following tools:
+| [compile](/reference/commands/compile) | Compiles (but does not run) the models in a project | All tools <br /> All [supported versions](/docs/dbt-versions/core) | Read | ✅ |
+| [debug](/reference/commands/debug) | Debugs dbt connections and projects | dbt Cloud IDE, dbt Core <br /> All [supported versions](/docs/dbt-versions/core) | Read | ✅ |
+| [deps](/reference/commands/deps) | Downloads dependencies for a project | All tools <br /> All [supported versions](/docs/dbt-versions/core) | Read | ✅ |
+| [docs](/reference/commands/cmd-docs) | Generates documentation for a project | All tools <br /> All [supported versions](/docs/dbt-versions/core) | Read | ✅ |
+| [environment](/reference/commands/dbt-environment) | Enables you to interact with your dbt Cloud environment. | dbt Cloud CLI <br /> Requires [dbt v1.5 or higher](/docs/dbt-versions/core) | N/A | N/A |
+| help | Displays help information for any command | dbt Core, dbt Cloud CLI <br /> All tools [supported versions](/docs/dbt-versions/core) | N/A | N/A |
+| [init](/reference/commands/init) | Initializes a new dbt project | dbt Core<br /> All [supported versions](/docs/dbt-versions/core) | Read | ✅ |
+| [list](/reference/commands/list) | Lists resources defined in a dbt project | All tools <br /> All [supported versions](/docs/dbt-versions/core) | Read | ✅ |
+| [parse](/reference/commands/parse) | Parses a project and writes detailed timing info | All tools <br /> All [supported versions](/docs/dbt-versions/core) | Read | ✅ |
+| reattach | Reattaches to the most recent invocation to retrieve logs and artifacts. | dbt Cloud CLI <br /> Requires [dbt v1.6 or higher](/docs/dbt-versions/core) | N/A | N/A |
+| [retry](/reference/commands/retry) | Retry the last run `dbt` command from the point of failure | All tools <br /> Requires [dbt v1.6 or higher](/docs/dbt-versions/core) | Write | ❌ |
+| [run](/reference/commands/run) | Runs the models in a project | All tools <br /> All [supported versions](/docs/dbt-versions/core) | Write | ❌ |
+| [run-operation](/reference/commands/run-operation) | Invoke a macro, including running arbitrary maintenance SQL against the database | All tools <br /> All [supported versions](/docs/dbt-versions/core) | Write | ❌ |
+| [seed](/reference/commands/seed) | Loads CSV files into the database | All tools <br /> All [supported versions](/docs/dbt-versions/core) | Write | ❌ |
+| [show](/reference/commands/show) | Preview table rows post-transformation | All tools <br /> All [supported versions](/docs/dbt-versions/core) | Read | ✅ |
+| [snapshot](/reference/commands/snapshot) | Executes "snapshot" jobs defined in a project | All tools <br /> All [supported versions](/docs/dbt-versions/core) | Write | ❌ |
+| [source](/reference/commands/source) | Provides tools for working with source data (including validating that sources are "fresh") | All tools<br /> All [supported versions](/docs/dbt-versions/core) | Read | ✅ |
+| [test](/reference/commands/test)
+
```

Comment:
What about the following?


The following sections outline the commands supported by dbt and their relevant flags. They are available in all tools and all [supported versions](https://docs.getdbt.com/docs/dbt-versions/core) unless noted otherwise.

|    Command    |                                         Description                                         | Parallel execution with other commands |                Caveats                |
|:-------------:|:-------------------------------------------------------------------------------------------:|:--------------------------------------:|:-------------------------------------:|
| build         | Build and test all selected resources (models, seeds, snapshots, tests)                     | ❌                                      |                                       |
| cancel        | Cancels the most recent invocation.                                                         | N/A                                    | dbt Cloud CLI only dbt v1.6 or higher |
| clean         | Deletes artifacts present in the dbt project                                                | ✅                                      |                                       |
| clone         | Clone selected models from the specified state                                              | ❌                                      | dbt v1.6 or higher                    |
| compile       | Compiles (but does not run) the models in a project                                         | ✅                                      |                                       |
| debug         | Debugs dbt connections and projects                                                         | ✅                                      | dbt Cloud IDE, dbt Core only          |
| deps          | Downloads dependencies for a project                                                        | ✅                                      |                                       |
| docs          | Generates documentation for a project                                                       | ✅                                      |                                       |
| environment   | Enables you to interact with your dbt Cloud environment.                                    | N/A                                    | dbt Cloud CLI only dbt v1.5 or higher |
| help          | Displays help information for any command                                                   | N/A                                    |                                       |
| init          | Initializes a new dbt project                                                               | ✅                                      | dbt Core only                         |
| list          | Lists resources defined in a dbt project                                                    | ✅                                      |                                       |
| parse         | Parses a project and writes detailed timing info                                            | ✅                                      |                                       |
| reattach      | Reattaches to the most recent invocation to retrieve logs and artifacts.                    | N/A                                    | dbt Cloud CLI only dbt v1.6 or higher |
| retry         | Retry the last run dbt command from the point of failure                                    | ❌                                      | dbt v1.6 or higher                    |
| run           | Runs the models in a project                                                                | ❌                                      |                                       |
| run-operation | Invoke a macro, including running arbitrary maintenance SQL against the database            | ❌                                      |                                       |
| seed          | Loads CSV files into the database                                                           | ❌                                      |                                       |
| show          | Preview table rows post-transformation                                                      | ✅                                      |                                       |
| snapshot      | Executes "snapshot" jobs defined in a project                                               | ❌                                      |                                       |
| source        | Provides tools for working with source data (including validating that sources are "fresh") | ✅                                      |                                       |